.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - "stable"
+cache:
+  directories:
+  - node_modules
+script:
+  - npm run build
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $github_token
```

Comment:
Do we have a `github_token`?

Reply:
or does this populate automatically

Reply:
Yeah, it's in travis settings since it's a GitHub access token that has write access to master

Reply:
Great

Reply:
we used an env variable so the token isn't public